package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.1",
   "license": "MIT",
   "scripts": {
+    "prestart": "cd packages/components && npm run build:cssTypes",
```

Comment:
🤯 I had no idea there were so many of these! https://docs.npmjs.com/misc/scripts

Reply:
I need to learn more `npm` 😆